rero_ils/modules/acq_receipt_lines/listener.py

```diff
@@ -40,3 +40,6 @@ def enrich_acq_receipt_line_data(sender, json=None, record=None, index=None,
             pid=record.order_line.account_pid,
             type='acac'
         )
+
+        json['total_amount'] = round(
+            record.amount*record.receipt.exchange_rate*record.quantity, 2)
```

Comment:
My feeling is to place the least as possible logic into listener.
Maybe add a `@property` method into the `AcqReceiptLine` and call it to fill `json['total_amount']`